package.json

```diff
@@ -47,6 +47,7 @@
   },
   "dependencies": {
     "chartjs-color": "^2.0.0",
-    "moment": "^2.10.6"
+    "moment": "^2.10.6",
+    "element-resize-detector": "^1.1.10"
```

Comment:
Is there any downside to using this library? I think I recall looking at the internals once and noticing that it does some light polling to debounce events to a single `requestAnimationFrame`. Have you seen any downsides from using this?